main.py

```diff
@@ -1,7 +1,34 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+
+    for genre_name in genres:
+        Genre.objects.get_or_create(name=genre_name)
+    for actor_first_name, actor_second_name in actors:
+        Actor.objects.get_or_create(
+            first_name=actor_first_name,
+            last_name=actor_second_name
+        )
+    Genre.objects.filter(name="Dramma").update(name="Drama")
```

Comment:
Would make sense to break down main function is smaller specialized functions to structure your code better. E.g separate function for creating, updating, deleting..